lib/content/utils.js

```diff
@@ -69,6 +69,118 @@ const check = {
   },
 };
 
+/**
+ * Grab type from all content types in system
+ *
+ * @param {object} req - Express Request Object
+ *
+ * @returns {promise} reject with error message; resolve with single content-type object
+ */
+const type = (req) => {
+  return new Promise((res, rej) => {
+    const typ = req.params.type.toString();
+
+    if (!typ || typ === '') {
+      rej(config.content.messages.missing.type.replace('%type', '(no type given)'));
+    }
+
+    const exists = req.content.types.find(tp => {
+      return tp.id === typ.toLowerCase();
+    });
+
+    if (!exists || (exists === false) || (typeof exists !== 'object') || !exists.id || !exists.name) {
```

Comment:
This feels… ugly. Can this if statement be cleaned up? For instance, `typeof exists !== 'object` will catch `!exists` and `exists === false`